CRM/Core/SelectValues.php

```diff
@@ -223,6 +223,7 @@ public static function customGroupExtends() {
       'ContributionRecur' => ts('Recurring Contributions'),
       'Group' => ts('Groups'),
       'Membership' => ts('Memberships'),
+      'MembershipType' => ts('Membership Types'),
```

Comment:
not needed

Reply:
@eileenmcnaughton The problem here is that you need to set 3 parameters (label, value, name) in cg_extend_objects but you cannot do this via the UI as name is not exposed.  So you are either limited to doing the DB change via an extension or editing the DB directly.  It may also be possible to do it via the API but it seems a lot less straightforward than just adding it here which allows the user to directly select "Membership Types" as a custom data entity when creating custom fields.
And we now have support in core for displaying those custom fields.

Reply:
@mattwire right - I think the intent IS that additional entities would be supported by adding values to that option group in an extension.